core/simd/simd.odin

```diff
@@ -2510,3 +2510,457 @@ Example:
 recip :: #force_inline proc "contextless" (v: $T/#simd[$LANES]$E) -> T where intrinsics.type_is_float(E) {
 	return T(1) / v
 }
+
+/*
+Creates a vector where each lane contains the index of that lane.
```

Comment:
The heading line of the doc should start with an imperative verb, just like they do with git commit messages.

```suggestion
Create a vector where each lane contains the index of that lane.
```